packages/sx.js/src/strategies/offchain/remote-vp.ts

```diff
@@ -20,9 +20,9 @@ export default function createRemoteVpStrategy(): Strategy {
 
```

Comment:
Can we add changelog entry for this?

Reply:
Done, sorry for the miss. Is there a github actions to remind of that ?